.github/workflows/cintegration-s3-apps-caller.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy to development environment
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches:
+      - main
+      - master
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# This workflow is made up of one job that calls the reusable workflow in graasp-deploy
+jobs:
+  graasp-deploy-s3-apps-workflow:
+    # todo: replace me
+    name: \#Graasp Template App\#
```

Comment:
Why with `#` on each side? Here and in other files, I've seen that.

Reply:
I would like to replace these by the name of the app when clonning with the cli. This is the pattern i will be matching.